### DISCUSSION TITLE
dom_designer/12.0.2/basic/H_DELETEUSER_METHOD_ADMINP

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#152] [Q&A]
# dom_designer/12.0.2/basic/H_DELETEUSER_METHOD_ADMINP

Enters a request in the Administration Requests database to delete a user.

https://help.hcl-software.com/dom_designer/12.0.2/basic/H_DELETEUSER_METHOD_ADMINP.html

**Comment by ShigemitsuTanaka** [comment]
The description of the %idvaultaction% parameter is wrong.
The explanation of 0 and 2 is reversed.
Please correct.
